src/db.cpp

```diff
@@ -5178,13 +5162,8 @@ void load_consist(void)
 
             snprintf(buf, sizeof(buf), "%s/Inside", sect_name);
             inside = data.GetInt(buf, 0);
+            // Lookup macro definition
             if (house_version == VERSION_HOUSE_FILE) {
-              // Since we're now saved the obj linked lists  in reverse order, in order to fix the stupid reordering on
```

Comment:
As long as these comments are accurate, they should be kept, as they're a good reference to what the code is doing. I'm always in favor of more verbose commenting.

Reply:
They were repetitive saying the same thing so I thought directing the documentation to the header was cleaner. Can add them back.

Reply:
Yes please. I think the ideal end state would be shifting all of the reproduced saving logic out into a centralized function that we can reuse in multiple places, but for now duplicating the documentation across the separate functions will help.

Reply:
Roger that!